static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -0,0 +1,327 @@
+          <StyledHeader key={1}>
+            {t('Recent Commits')}{' '}
+            <Tooltip title={t('Based on the last 30 days of commit data')}>
+              <IconInfo size="xs" />
+            </Tooltip>
+          </StyledHeader>,
+          t('Role'),
+          t('Team'),
+        ]}
+      >
+        {members?.map((member, i) => {
+          const checked = members[i].selected;
+          return (
+            <Fragment key={i}>
+              <div>
+                <Checkbox
+                  aria-label={t('Select %s', member.email)}
+                  checked={checked}
+                  onChange={() => checkboxToggle(!checked, i)}
+                />
+              </div>
+              <StyledPanelItem>
+                <ContentRow>
+                  <IconGithub size="sm" />
+                  <StyledExternalLink href={`http://github.com/${member.externalId}`}>
+                    {tct('@[userId]', {userId: member.externalId})}
+                  </StyledExternalLink>
+                </ContentRow>
+                <Subtitle>{member.email}</Subtitle>
+              </StyledPanelItem>
+              <ContentRow>
+                <IconCommit size="sm" />
+                {member.commitCount}
+              </ContentRow>
+              <RoleSelectControl
+                aria-label={t('Role')}
+                data-test-id="select-role"
+                disabled={false}
+                roles={invitableRoles}
+                disableUnallowed
+                onChange={value => setRole(value?.value, i)}
+              />
+              <TeamSelector
+                aria-label={t('Add to Team')}
+                data-test-id="select-teams"
+                disabled={false}
+                placeholder={t('Add to teams\u2026')}
+                onChange={opts => setTeams(opts ? opts.map(v => v.value) : [], i)}
+                multiple
+                clearable
+              />
+            </Fragment>
+          );
+        })}
+      </StyledPanelTable>
+      <Footer>
+        <div>{renderStatusMessage()}</div>
+        <ButtonBar gap={1}>
+          <Button size="sm" onClick={() => closeModal()} style={{marginRight: space(1)}}>
+            {t('Cancel')}
+          </Button>
+          <Button
+            size="sm"
+            priority="primary"
+            aria-label={t('Send Invites')}
+            onClick={sendInvites}
+            style={{marginRight: space(1)}}
+            disabled={!canSend || selectedCount === 0}
+          >
+            {inviteButtonLabel()}
+          </Button>
+        </ButtonBar>
+      </Footer>
+    </Fragment>
+  );
+
+  return (
+    <InviteModalHook
+      organization={organization}
+      willInvite
+      onSendInvites={sendMemberInvites}
+    >
+      {hookRenderer}
+    </InviteModalHook>
+  );
+}
+
+export default InviteMissingMembersModal;
+
+const StyledPanelTable = styled(PanelTable)`
+  grid-template-columns: max-content 1fr max-content 1fr 1fr;
+  overflow: visible;
```

Comment:
is `overflow: visible` necessary here?

Reply:
i need this so that the dropdowns are visible over the contents of the other rows / columns. otherwise they look like this lol
<img width="805" alt="Screenshot 2023-08-17 at 15 38 09" src="https://github.com/getsentry/sentry/assets/70817427/ce631cf3-75bf-4448-b0d6-a99bb00e29f5">

Reply:
oh gotcha that makes sense